ipaserver/install/cainstance.py

```diff
@@ -1277,6 +1279,15 @@ def __enable_instance(self):
             config = []
         self.ldap_configure('CA', self.fqdn, None, basedn, config)
 
+
+    def __expose_ca_in_ldap(self):
```

Comment:
Could you please add a comment that this is a hack for DL0 and will be removed once we remove DL0 code?

Reply:
Sure